mpd/client.go

```diff
@@ -591,6 +591,76 @@ func (c *Client) List(uri string) ([]string, error) {
 	return ret, nil
 }
 
+// Output related commands.
+
+// ListOutputs lists all configured outputs with their name, id & enabled state.
+func (c *Client) ListOutputs() ([]map[string]interface{}, error) {
```

Comment:
This package doesn't attempt to parse any of the attributes in the outputs. One can argue for parsing them, but then it would be inconsistent with the rest of the package. So, I think this function should be:
```
func (c *Client) ListOutputs() ([]Attrs, error)
```
See how Client.ListPlaylists is implemented using Client.readAttrsList. You just need a call to c.readAttrsList("outputid").